paasta_tools/utils.py

```diff
@@ -1942,16 +1944,17 @@ class SystemPaastaConfigDict(TypedDict, total=False):
     log_writer: LogWriterConfig
     maintenance_resource_reservation_enabled: bool
     marathon_servers: List[MarathonConfigDict]
-    mark_for_deployment_max_polling_threads: int
-    mark_for_deployment_default_polling_interval: float
-    mark_for_deployment_default_diagnosis_interval: float
     mark_for_deployment_default_default_time_before_first_diagnosis: float
+    mark_for_deployment_default_diagnosis_interval: float
+    mark_for_deployment_default_polling_interval: float
+    mark_for_deployment_max_polling_threads: int
     mark_for_deployment_should_ping_for_unhealthy_pods: bool
     mesos_config: Dict
     metrics_provider: str
     monitoring_config: Dict
     nerve_readiness_check_script: List[str]
     nerve_register_k8s_terminating: bool
+    paasta_api_number_workers: int
```

Comment:
this is the only new change here - i just sorted what seems to have originally been a sorted list until we stopped caring :p